### ISSUE TITLE
ui: clarify replicaIDs in /debug/range 

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#16583]
Observe that in this picture (which is not from current `master`, but I just checked and it looks the same there) the "lease holder", "vote" and "raft" cells are hard to parse. I think they contain ReplicaIDs, but the ReplicaID isn't shown until the very bottom of the table and you're left to guess it from context (and users are likely to clip that lookup table from the screenshot, like in this case). Instead of `Vote: 4` I'd prefer reading something like `n2:s2/replicaID4` directly.

![image](https://user-images.githubusercontent.com/5076964/27255337-60b16096-536a-11e7-9d16-287c86632a93.png)

**Comment by dianasaur323** [comment]
@BramGruneir Do you mind taking a look? I think you implemented these pages?

**Comment by BramGruneir** [comment]
Yep, it's an easy fix.  I've added it to the queue.  If I don't get to it right away, I'll be sure to fix it as we port these pages to the admin ui.

**Comment by dianasaur323** [comment]
Great, thanks Bram!
